homeassistant/components/climate/fritzhome.py

```diff
@@ -0,0 +1,137 @@
+"""
+Support for AVM Fritz!Box fritzhome thermostate devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/climate.fritzhome/
+"""
+import logging
+
+from homeassistant.components.fritzhome import DOMAIN
+from homeassistant.components.climate import (
+    ATTR_OPERATION_MODE, ClimateDevice, STATE_ECO,
+    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE)
+from homeassistant.const import (PRECISION_HALVES)
+from homeassistant.const import (TEMP_CELSIUS, ATTR_TEMPERATURE)
+
+DEPENDENCIES = ['fritzhome']
+
+_LOGGER = logging.getLogger(__name__)
+
+STATE_COMFORT = 'comfort'
+STATE_MANUAL = 'manual'
+
+SUPPORT_FLAGS = (SUPPORT_TARGET_TEMPERATURE | SUPPORT_OPERATION_MODE)
+
+OPERATION_LIST = [STATE_COMFORT, STATE_ECO, STATE_MANUAL]
```

Comment:
This list should only return states that are defined in the base climate component. You're not allowed to define custom device specific states, that are user facing.

Reply:
Can i add these operation modes to the climate base component? Otherwise I have a problem to map these modes to currently availables.

Reply:
What do you think @tinloaf?

Reply:
I would be fine with it; however, I'd also ask you (@hthiery , I mean) not to be offended if we change this a bit when we do some refactoring in the future. Names may change because we will try to consolidate constants across platforms.

Reply:
I'm fine with that until there is mode that matches the purpose. I would like to change the comfort mode in the fritzhome thermostate to use the `STATE_HEAT` instead of `STATE_COMFORT` and add `STATE_MANUAL` to the base climate component.